objects_count:
  chair: inf
  pottedplant: 1
  diningtable: 1